src/data/bucket/symbol_bucket.js

```diff
@@ -414,16 +418,28 @@ class SymbolBucket implements Bucket {
         }
     }
 
+    update(states: FeatureStates, vtLayer: VectorTileLayer) {
```

Comment:
I would assume that a method called `Bucket#update` did something more general than this (like update all the data in the bucket). Would a name like `Bucket#updateFeatureState` be more clear?

Reply:
`Bucket#update` updates the dynamic buffers in the bucket with the passed in `states`. The layout and index buffers are static, and so cannot be updated. The more specific name is more descriptive, but it doesn't distinguish itself from any other type of `update`.